comm.go

```diff
@@ -122,3 +122,22 @@ func rpcWithControl(msgs []*pb.Message,
 		},
 	}
 }
+
+func copyRPC(rpc *RPC) *RPC {
+	res := &RPC{
```

Comment:
```go
res := new(RPC)
*res = *rpc
*res.Control = *rpc.Control
```

Reply:
Hrm, we still need to create the control object -- it's a pointer.

Reply:
But still, more idiomatic and terse.